Feature/enemy model

Aesnas opened the pull request:
- Added Low Poly Enemy Model v2, (UV'd, not rigged)
- Added textures to be imported into Substance Painter
- Replaced EnemyPlayer prefab

Aesnas committed:
Merge branch 'develop' into feature/enemy_model

Aesnas committed:
Merge branch 'develop' into feature/enemy_model

Aesnas committed:
Merge branch 'develop' into feature/enemy_model

Aesnas committed:
Merge branch 'develop' into feature/enemy_model

Aesnas committed:
Updated Low Poly Model Enemy

- Added Low Poly Enemy Model v2, (UV'd, not rigged)
- Added textures to be imported into Substance Painter
- Replaced EnemyPlayer prefab

Calvin-Davidson reviewed (changes requested):
TestMat kan weg?, textures for substance?

Aesnas commented:
> TestMat kan weg?, textures for substance?

Ja TestMat kan weg.
Textures for substance is voor Joost om te implementeren in Substance Painter, die zou de enemy model texturen

Calvin-Davidson commented:
> > TestMat kan weg?, textures for substance?
> 
> Ja TestMat kan weg. Textures for substance is voor Joost om te implementeren in Substance Painter, die zou de enemy model texturen

Alright, dan laat ik hem even open staan, dit soort files willen we niet persee het project in pushen

joost18JKatGit committed:
added enemy textures

removed bake maps because I downloaded them, added textures and applied to enemy model (nr. 2). Changed materials because the metallic and roughness didn't look right. changed scaling from 1000x1000 to 1x1 for both materials.

Calvin-Davidson reviewed (changes requested):
TestMat.mat mag weg

Calvin-Davidson committed:
Update EnemyAI.prefab

Calvin-Davidson committed:
Removed test.mat

Calvin-Davidson committed:
Merge branch 'develop' into feature/enemy_model
